iStore/ProductDetails/AddToCardButton.swift

```diff
@@ -8,9 +8,12 @@
 import SwiftUI
 
 struct AddToCardButton: View {
+    
+    let action: () -> Void
```

Comment:
Why public?

Reply:
because if private it won't be accessible from the cartView.